File: app/src/main/java/Model/FirebaseUser.java
package Model;

import java.util.List;

public class FirebaseUser extends User {

    private List<Item> cartItems;
    private List<Item> wishList;


    public List<Item> getCartItems() {
        return cartItems;
    }

    public void setCartItems(List<Item> cartItems) {
        this.cartItems = cartItems;
    }

    public List<Item> getWishList() {
        return wishList;
    }

    public void setWishList(List<Item> wishList) {
        this.wishList = wishList;
    }
}

File: app/src/main/java/com/whiteside/cafe/model/FirebaseUser.kt
package com.whiteside.cafe.model

class FirebaseUser : User() {
    private var cartItems: MutableList<Item?>? = null
    private var wishList: MutableList<Item?>? = null
    fun getCartItems(): MutableList<Item?>? {
        return cartItems
    }

    fun setCartItems(cartItems: MutableList<Item?>?) {
        this.cartItems = cartItems
    }

    fun getWishList(): MutableList<Item?>? {
        return wishList
    }

    fun setWishList(wishList: MutableList<Item?>?) {
        this.wishList = wishList
    }
}